.github/actions/pr-labeler/README.md

```diff
@@ -0,0 +1,18 @@
+# PR Labeler Action
+
+This is a standalone github action hosted within wc-admin that supports
+auto adding or removing a PR label, which could be useful in various
+automation scenarios.
+
+To keep it simple and avoid checking in `node_modules` when you make
+changes to `index.js` you'll need to compile it into the `dist` directory
+via [@vercel/ncc](https://github.com/vercel/ncc).
```

Comment:
To host an action in our repo, this was the simplest way, I've included some basic instructions for the next person who comes along. If we need to make this more automated or fancy we can do so later.